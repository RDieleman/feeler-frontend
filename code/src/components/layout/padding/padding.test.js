import React from "react";
import {shallow} from "enzyme";
import {PaddingComponent} from "./padding.component";

it("Expect to render Padding component", () => {

    expect(shallow(
        <PaddingComponent

        />)).toMatchSnapshot();
});