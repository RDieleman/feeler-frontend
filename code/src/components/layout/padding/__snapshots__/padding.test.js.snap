// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Padding component 1`] = `ShallowWrapper {}`;
